packages/ag-charts-enterprise/src/test/load-example.ts

```diff
@@ -0,0 +1,46 @@
+import * as agCharts from '../main';
+
+const filters = [
+    /.*AgChart\.(update|create)/,
+    /.*AgEnterpriseCharts\.(update|create)/,
+    /.* container: .*/,
+    /.*setInterval.*/,
+    /.*setTimeout.*/,
+    /^Object\.defineProperty/,
+    /^exports\./,
+    /^const ag_charts_community_1 =/,
+    /^const ag_charts_enterprise_1 =/,
+    /^const data_1 =/,
+];
+
+const cleanJs = (content: Buffer | string) => {
+    const inputLines = (typeof content === 'string' ? content : content.toString()).split('\n');
+    const lines: string[] = [];
+
+    for (let line of inputLines) {
+        // Remove grossly unsupported lines.
+        if (filters.some((f) => f.test(line))) continue;
+        // Remove declares.
+        line = line.replace(/declare var.*;/g, '');
+
+        lines.push(line);
+    }
+
+    return lines;
+};
```

Comment:
Could we reference the versions of these functions in `ag-charts-community` rather than repeating them here?